helm_deploy/hmpps-incident-reporting-api/values.yaml

```diff
@@ -41,7 +43,7 @@ generic-service:
           return 401;
         }
   env:
-    JAVA_OPTS: "-Xmx3g"
+    JAVA_OPTS: "-Xmx4g"
```

Comment:
you could probably go a bit higher given the k8s limit is now 6GB but let's see how this goes.